packages/react-scripts/config/env.js

```diff
@@ -6,20 +6,35 @@
  * LICENSE file in the root directory of this source tree.
  */
 // @remove-on-eject-end
-'use strict';
+'use strict'
 
-const fs = require('fs');
-const path = require('path');
-const paths = require('./paths');
+const fs = require('fs')
+const path = require('path')
+const paths = require('./paths')
+
+let zionUiVersion
+try {
+  const resolvedZionUiPath = require.resolve('@fs/zion-ui')
+  const zionUiPackageJson = require(`${resolvedZionUiPath.split('@fs/zion-ui')[0]}@fs/zion-ui/package.json`)
+  zionUiVersion = zionUiPackageJson.version
+} catch (error) {
+  zionUiVersion = '-'
+}
+
+let reactScriptsVersion
+try {
+  const reactScriptsPackageJson = require('../package.json')
+  reactScriptsVersion = reactScriptsPackageJson.version
+} catch (error) {
+  reactScriptsVersion = '-'
+}
```

Comment:
Here is the logic for finding the zion-ui version and react-scripts version. Wrapped in a try/catch just in case something weird happens and it doesn't find it. The version will be defaulted to '-'.

Reply:
Should we log a sentry error if an error occurs so we can tell when it's not working?

Reply:
I don't think we have anything configured for logging to sentry during the build step...I'm sure we could figure it out, but it just isn't configured or setup.

Reply:
And it'll be obvious when it errors out, the analytics will say the stack is '-'. :)